nums/numpy/api/la.py

```diff
@@ -0,0 +1,299 @@
+# coding=utf-8
+# Copyright (C) 2020 NumS Development Team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# pylint: disable = redefined-builtin, too-many-lines, W1401, W0401, W0614
+
+from nums.core.application_manager import instance as _instance
+from nums.core.array.blockarray import BlockArray
+
+from nums.numpy.api.creation import *
+from nums.numpy.api.generated import *
+from nums.numpy.api.utility import *
+
+############################################
+# Linear Algebra Ops
+############################################
+
+
+def tensordot(x1: BlockArray, x2: BlockArray, axes=2) -> BlockArray:
+    """Compute tensor dot product along specified axes.
+
+    This docstring was copied from numpy.tensordot.
+
+    Some inconsistencies with the NumS version may exist.
```

Comment:
Idea: we could probably put this in one of those warning boxes. As we verify docstrings for accuracy, we can remove the warning message. In any case, we could do this in a follow up PR.

Reply:
Do you know how to do that within a docstring? The documentation is parsing everything purely from docstrings.